tools/github/scripts/Add-AzsdkMilestones.ps1

```diff
@@ -68,8 +68,13 @@ if (!$milestones) {
     return
 }
 
+$completed = 0
+$total = $Repositories.Length * $milestones.Length
+Write-Progress -Activity ($activity = 'Creating milestones') -PercentComplete 0
+
 foreach ($repo in $Repositories) {
     foreach ($m in $milestones) {
+        Write-Progress -Activity $activity -Status "In $repo" -CurrentOperation "Milestone $($m.Title)" -PercentComplete ($completed / $total * 100)
```

Comment:
Nice timing.   I just added something similar for the label sync.  Adjusted to steal some ideas from here.